src/alt/index.js

```diff
@@ -27,44 +30,6 @@ class Alt {
     this.batchingFunction(() => this.dispatcher.dispatch({ action, data, details }))
   }
 
-  createUnsavedStore(StoreModel, ...args) {
-    const key = StoreModel.displayName || ''
-    store.createStoreConfig(this.config, StoreModel)
-    const Store = store.transformStore(this.storeTransforms, StoreModel)
-
-    return fn.isFunction(Store)
-      ? store.createStoreFromClass(this, Store, key, ...args)
-      : store.createStoreFromObject(this, Store, key)
-  }
-
-  createStore(StoreModel, iden, ...args) {
```

Comment:
Probably need to keep this around with a deprecation warning to make it easier for users to upgrade.